route53.tf

```diff
@@ -45,3 +45,15 @@ resource "aws_route53_record" "default" {
     evaluate_target_health = true
   }
 }
+
+resource "aws_route53_record" "traefik" {
+  zone_id = aws_route53_zone.default.zone_id
+  name    = var.domain
+  type    = "A"
+
+  alias {
+    name                   = module.nlb.lb_dns_name
+    zone_id                = module.nlb.lb_zone_id
+    evaluate_target_health = true
+  }
+}
```

Comment:
When we have a wildcard record already, why should we make use of this? I'd think there's no need to host something directly on the "root" record of the domain.

Reply:
i know. But why would you want to do that instead of using something like `traefik.${var.domain}`?

Reply:
I want to keep a line with the default traefik config. On the other side, it makes it easy to test for future traefik implementations like tmp alb replacements.